src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,16 @@
-export const UserInfo = () => {};
+import { User } from '../types/User';
+
+type Props = {
+  user: User | null,
```

Comment:
```suggestion
  user: User,
```
let's render this component only when user exists